hack/operatorhub/README.md

```diff
@@ -0,0 +1,17 @@
+Operator Hub Release Generator
+===============================
+
+Extracts CRDs and RBAC definitions from `all-in-one.yaml` and generates the files required to publish a new release to Operator Hub.
```

Comment:
It may also be worth nothing that we still need to update `elastic-cloud-eck.package.yaml` as well. And maybe also the examples of how to test the generated config (`make operator.test OP_PATH=upstream-community-operators/elastic-cloud`)? It would be nice as a release manager to just come here and have the readme tell me everything I need to do.

Reply:
The package file is generated by the tool as well. I plan to update the release checklist to point to this tool so hopefully that addresses your second concern.

Reply:
Ah I switched dirs right into the 1.1.2 dir 🤦‍♀️ . and yep having it in the release checklist works for me, though I still think having instructions how to run the test/cleanup directives is useful so you don't need to rummage through the operator hub docs